pom.xml

```diff
@@ -75,19 +76,25 @@
                 <type>pom</type>
                 <scope>import</scope>
             </dependency>
+
+            <!-- need to override spring-core to fix  CVE-2025-41249(8.699999809265137) -->
+            <dependency>
+                <groupId>org.springframework</groupId>
+                <artifactId>spring-core</artifactId>
+                <version>6.2.11</version>
+            </dependency>
         </dependencies>
     </dependencyManagement>
 
     <dependencies>
+        <dependency>
```

Comment:
I think this needs a comment, naming which CVE it fixes.

Reply:
it's not a fix for a cve, it has been added so we don't rely on a transitive dependency. That is bad practice. It was previously missing as it was brought in by another library that also uses it.

Reply:
https://softwareengineering.stackexchange.com/questions/320128/is-it-better-to-rely-on-transitive-dependencies-or-explicitly-declare-them

Reply:
All clear now.